### PR TITLE
bump calisp

### PR DESCRIPTION
**Pull request by bernt-matthias**
and add script for filtering and peptide summary based on https://github.com/kinestetika/Calisp/blob/master/benchmarking/sif-benchmarking.ipynb

TODO: 

- [x] clarify TODO in python script
- [x] make outputs optional
- [x] update help

### CODE REVIEW

**Comment by bernt-matthias** [comment]
Hey @debjyoti197 can you approve the PR?

**Review by debjyoti197 — APPROVED** [review]

**Comment by debjyoti197** [comment]
@bernt-matthias approved!

File: tools/calisp/calisp.xml [hunk 1]
```diff
@@ -30,8 +30,33 @@ calisp
...
+#if $isotope_abundance_matrix
+    && ISOTOPE_ABUNDANCE_MATRIX="$isotope_abundance_matrix"
+#else
+    && ISOTOPE_ABUNDANCE_MATRIX="\$(python -c 'import site; print(f"{site.getsitepackages()[0]}/calisp/isotope_matrix.txt")')"
+#end if
+## row numbers in the factor matrix
+## C=1 N=2 O=3 H=4 S=5
+#set rowcol = {"13C": (1, 2), "14C": (1, 3), "15N": (2, 2), "17O": (3, 2), "18O": (3, 3), "2H": (4, 2), "3H": (4, 3), "33S": (5, 2), "34S": (5, 3), "36S": (5, 4)}
+#set selected_isotope = str($isotope)
+&& FACTOR=\$(grep -v "^#" "\$ISOTOPE_ABUNDANCE_MATRIX" | sed 's/ \+/\t/g'| cut -f$rowcol[str($selected_isotope)][1] | head -n $rowcol[$selected_isotope][0] | tail -n 1)
```
Comment (bgruening) [inline tools/calisp/calisp.xml:47]: Why can this not be done in Python?

**Review by bgruening — APPROVED** [review]
I would move the complexity from bash to python, but I trust you that there are other reasons.

File: tools/calisp/calisp.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/calisp/calisp.xml:47]: Indeed we could do this...

**Comment by bernt-matthias** [comment]
> So I commented tests again.

They were successful locally.

**Comment by bgruening** [comment]
Merge it and proceed I guess!

**Review by bgruening — APPROVED** [review]

**Pull request merged**